### ISSUE TITLE
Drifted versions in "last X versions"

### ISSUE THREAD

**Issue by eligao** [browserslist/browserslist#439]
`caniuse-db` updated their data to rev `1.0.30001022` ( https://github.com/Fyrd/caniuse/commit/a133838cd287d3e807151a4c39ebead7bb3b18b6 ) right after the commit https://github.com/browserslist/browserslist/commit/1e61852c55a31c83520392a61e3d2ad7d6cd0e8c in the issue https://github.com/browserslist/browserslist/issues/437, and the results are drifted instead.
For example, the query `last 2 safari versions` returns 
```
safari 13
safari TP
```
instead of 
```
safari 13
safari 12.1
```
The same applies to most if not all other browsers like Chrome or IE, etc.
I've created a PR to revert it: https://github.com/browserslist/browserslist/pull/438
FYI this is the latest agent data from `caniuse-lite@1.0.30001022`: https://gist.github.com/eligao/56c2d1938d5fd9875262c0c37cc9e33b
And this is the minimal reproducible example: https://codesandbox.io/s/sweet-fog-1xe6e

**Comment by XhmikosR** [comment]
I saw a huge bump in our CSS prefixes after switching to v4.8.4 too. [bundlesize link](https://bundlesize-store.now.sh/build?info=%7B%22files%22%3A%5B%7B%22maxSize%22%3A7168%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap-grid.css%22%2C%22size%22%3A7821%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A6400%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap-grid.min.css%22%2C%22size%22%3A6699%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A2048%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap-reboot.css%22%2C%22size%22%3A1715%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A2048%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap-reboot.min.css%22%2C%22size%22%3A1609%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A25856%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap.css%22%2C%22size%22%3A27907%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A23808%2C%22path%22%3A%22.%2Fdist%2Fcss%2Fbootstrap.min.css%22%2C%22size%22%3A25448%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A48640%2C%22path%22%3A%22.%2Fdist%2Fjs%2Fbootstrap.bundle.js%22%2C%22size%22%3A48311%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A22784%2C%22path%22%3A%22.%2Fdist%2Fjs%2Fbootstrap.bundle.min.js%22%2C%22size%22%3A22588%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A25600%2C%22path%22%3A%22.%2Fdist%2Fjs%2Fbootstrap.js%22%2C%22size%22%3A25309%2C%22compression%22%3A%22gzip%22%7D%2C%7B%22maxSize%22%3A15872%2C%22path%22%3A%22.%2Fdist%2Fjs%2Fbootstrap.min.js%22%2C%22size%22%3A15735%2C%22compression%22%3A%22gzip%22%7D%5D%2C%22repo%22%3A%22twbs%2Fbootstrap%22%2C%22branch%22%3A%22v4-dev-xmr-drop-node-6%22%2C%22commit_message%22%3A%22%22%2C%22sha%22%3A%22c8679b89e05eb01255ae68e5f815540e67c2720e%22%7D)

**Comment by eligao** [comment]
It is also breaking `https://github.com/amilajack/eslint-plugin-compat` because of the "Safari TP" version matched with this rule. Which is actually the reason why I dug it all the way here.

**Comment by vith** [comment]
Tracked this problem here from @babel/preset-env leaving the optional chaining operator un-transpiled when targeting "last 1 chrome version". That query is resolving to chrome 80, which is not yet released.

**Comment by equinusocio** [comment]
Same issue with "last 1 Firefox versions" which return an error:

`"Unknown version 73 of firefox"`

**Comment by Dangoo** [comment]
Some additional testing uncovered a difference for queries passed as argument compared to queries defined in package.json:

Single query as argument:
```bash
$ npx browserslist "last 1 Safari major versions"
safari 13
```

Multiple queries as argument:
```bash
$ npx browserslist "last 1 Safari major versions, last 1 iOS major versions"
ios_saf 13.2
ios_saf 13.0-13.1
safari 13
```

Defined by `package.json`:
```bash
# package.json
# "browserslist": [
#   "last 1 Safari major versions",
#   "last 1 iOS major versions"
# ]

$ npx browserslist
ios_saf 13.3
ios_saf 13.2
ios_saf 13.0-13.1
safari 13
safari 12.1
safari 12
safari 11.1
safari 11
safari 10.1
safari 10
safari 9.1
safari 9
safari 8
safari 7.1
safari 7
safari 6.1
safari 6
safari 5.1
safari 5
safari 4
safari 3.2
safari 3.1
safari TP
```

**Comment by ai** [comment]
It seems like Can I Use data is inconsistency in terms of stable/unreleased releases. 

Sorry, the fix could take a day since hotfix will bring another issue.

**Comment by ai** [comment]
Fixed at 4.8.5

**Comment by ai** [comment]
If you want to support us, here is release tweet
https://twitter.com/Browserslist/status/1219725386340347905

Here is our Open Collective
https://opencollective.com/browserslist
